pkg/agent/flowexporter/connections/connections.go

```diff
@@ -0,0 +1,120 @@
+package connections
+
+import (
+	"strconv"
+	"sync"
+	"time"
+
+	"k8s.io/klog"
+
+	"github.com/vmware-tanzu/antrea/pkg/agent/flowexporter"
+	"github.com/vmware-tanzu/antrea/pkg/agent/interfacestore"
+	"github.com/vmware-tanzu/antrea/pkg/agent/openflow"
+)
+
+var _ CxnStoreAndBuilder = new(cxnStoreAndBuilder)
+
+type CxnStoreAndBuilder interface {
+	Run(stopCh <-chan struct{})
+}
+
+// TODO: Name dilemma--cxnStoreAndBuilder or cxnCache or something else?
```

Comment:
I would say that `connectionStore` (or maybe `connStore`) works for me. Do you think the `AndBuilder` is necessary?

Reply:
I wanted to get in the poller aspect in name. As poller was highlighted in other objects names and comment, simpler name is good. I picked `connectionStore `.